HISTORY.rst

```diff
@@ -2,6 +2,63 @@
+  (@JBorrow on GitHub) for the contribution.
+* Fix an issue where units would be printed twice in formatted strings with
+  an ``unyt_array`` embedded. See `PR #188
+  <https://github.com/yt-project/unyt/pull/180>`_. Thank you to Clément Robert
+  (@neutrinoceros on GitHub) for the contribution.
+* Add a method to parse a ``unyt_quantity`` from a string expression. See `PR #191
+  <https://github.com/yt-project/unyt/pull/191>`_. Thank you to Clément Robert
+  (@neutrinoceros on GitHub) for the contribution.
+* Fix an issue where a ``unyt_array`` with dtype int8 could not be converted
+  to a different unit. See `PR #197
+  <https://github.com/yt-project/unyt/pull/197>`_. Thank you to Clément Robert
+  (@neutrinoceros on GitHub) for the contribution.
+* The import time for ``unyt`` has been reduced. See `PR #199
+  <https://github.com/yt-project/unyt/pull/199>`_. Thank you to Clément Robert
+  (@neutrinoceros on GitHub) for the contribution.
+* Fix an issue where taking an ``unyt_array`` or ``unyt_quantity`` to a zero
+  power would retain the units of the original array or quantity instead of
+  converting to a dimensionless array. See `PR #204
+  <https://github.com/yt-project/unyt/pull/204>`_. Thank you to Josh Borrow
+  (@JBorrow on GitHub) for the contribution.
+* Add support for coercing iterables of ``unyt_array`` objects with nonuniform
+  dimensionally equivalent units to a single ``unyt_array``. See `PR #211
+  <https://github.com/yt-project/unyt/pull/211>`_. Thank you to Nathan Goldbaum
+  (@ngoldbaum on GitHub) for the contribution.
+* Add the civil engineering units ``pli``, ``plf``, ``psf``, ``kli``, ``klf``,
+  and ``ksf``. See `PR #217 <https://github.com/yt-project/unyt/pull/217>`_.
+  Thank you to @osnippet on GitHub for the contribution.
+* Fix typos in constants and unit prefixes. See `PR #218
+  <https://github.com/yt-project/unyt/pull/218>`_. Thank you to Clément Robert
+  (@neutrinoceros on GitHub) for the contribution.
+* Fix an issue where multiplying a 1-element ``unyt_array`` would return a
+  ``unyt_quantity``. See `PR #225 <https://github.com/yt-project/unyt/pull/225>`_.
+  Thank you to Clément Robert (@neutrinoceros on GitHub) for the contribution.
+* Add the Rydberg constant ``R_∞`` and unit ``Ry``, add the dimension
+  ``angular_frequency`` and the unit ``rpm``, and increase the precision of
+  Avogadro's number. See `PR #228 <https://github.com/yt-project/unyt/pull/228>`_.
+* Fix an issue where ``np.divide.reduce`` returns incorrect units for ``unyt_array``
+  instances. See `PR #230 <https://github.com/yt-project/unyt/pull/230>`_.
+  Thank you to Kyle Oman (@kyleaoman on GitHub) for the contribution.
+* Fix a regression where ``unyt_array.reshape`` wasn't wrapping
+  ``numpy.ndarray.reshape`` correctly. See `PR #242
+  <https://github.com/yt-project/unyt/pull/242>`_. Thank you to Clément Robert
+  (@neutrinoceros on GitHub) for the contribution.
```

Comment:
Not sure this is worth mentioning in the changelog: this regression was never released, so it's not a diff wrt to 2.8.0

Reply:
If it's ok with you, we can just remove it then